src/layouts/Engagement_Plan_Task__c-Action Plan Task Layout.layout

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@davidhabib I think this was added back in by mistake, there is an Engagement_Plan_Task__c-Engagement Plan Task Layout.layout